l10n_nl_dutch_company_type/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+[ This file must be max 2-3 paragraphs, and is required. ]
```

Comment:
Put some real description here. Maybe a link to a wiki or to  standard where the dutch company types are described.